cli/container.go

```diff
@@ -165,11 +158,8 @@ func removeContainer(ID string) error {
 	if err != nil {
 		return fmt.Errorf("ERROR: couldn't create docker client\n%+v", err)
 	}
-	if err := docker.ContainerRemove(ctx, ID, types.ContainerRemoveOptions{}); err != nil {
-		log.Printf("WARNING: couldn't delete container [%s], trying a force remove now.", ID)
-		if err := docker.ContainerRemove(ctx, ID, types.ContainerRemoveOptions{Force: true}); err != nil {
-			return fmt.Errorf("FAILURE: couldn't delete container [%s] -> %+v", ID, err)
-		}
+	if err := docker.ContainerRemove(ctx, ID, types.ContainerRemoveOptions{Force: true}); err != nil {
```

Comment:
Would there be any benefit in doing a `stop` then a graceful `rm` over doing `rm -f`?
I'm not sure about this and if not, I'd be super happy about removing those unnecessary log messages :+1:

Reply:
Not really.  But if you want to do the graceful stop you have to actually call the stop API.  Remove will just fail if a container is running and the Force will send a sigkill.  So I don't think what is currently on master actually will ever gracefully stop the container.

Reply:
It will only gracefully remove stopped containers, but I think that even that is not really necessary to fulfill the endgoal of the command.